File: Python/Calculadora.py
# -*- coding: utf-8 -*-
print("Bienvenue!")
print("+ Soma          | - Subtração")
print("* Multiplicação | / Divisão")
print("^ Potenciação   | v Radiciação")

op = input("Qual operação você deseja realizar hoje?: ")

if op == '+':
	a = float(input("\nInsira o valor inicial: "))
	b = float(input("Insira o valor a ser somado: "))
	c = a+b;
	a = str(a);b = str(b); c = str(c);
	print(a + " + " + b + " = " + c);
elif op == '-':
	a = float(input("\nInsira o valor inicial: "))
	b = float(input("Insira o valor a ser subtraido: "))
	c = a-b;
	a = str(a);b = str(b);c = str(c);
	print(a + " - " + b + " = " + c);
elif op == '*':
	a = float(input("\nInsira o fator inicial: "))
	b = float(input("Insira o segundo fator: "))
	c = a*b;a = str(a);b = str(b);c = str(c);
	print(a + " * " + b + " = " + c);
elif op == '/':
	a = float(input("Insira o dividendo: "))
	b = float(input("Insira o divisor: "))
	c = a/b;
	a = str(a);b = str(b);c = str(c);
	print(a + " / " + b + " = " + c);
elif op == '^':
	a = float(input("\nInsira o valor da base: "))
	b = float(input("Insira o exponente: "))
	c = a**b;
	a = str(a);b = str(b);c = str(c);
	print(a + " ^ " + b + " = " + c);
elif op == 'v':
	a = float(input("\nInsira o valor do radiciando: "))
	b = int(input("Insira o índice da raiz: "))
	c = a**(1/b);a = str(a);b = str(b);c = str(c);
	if b == '2':
		print("O valor da raiz quadrada de " + a + " é " + c);
	elif b == '3':
		print("O valor da raiz cúbica de " + a + " é " + c);
	else:
		print("O valor da raiz " + b + "a de " + a + " é " + c);
else:
	print("Insira um valor válido, obrigado")


print("Até mais!")

File: Ruby/Calculadora.rb
puts "Bienvenue! Qual operação você deseja realizar?"
puts"-----------------------------------------------"
puts "+ Soma          | - Subtração"
puts "* Multiplicação | / Divisão"
puts "^ Exponenciação | v Radiciação"
puts"-----------------------------------------------"
op = gets.chomp
repeat = 1
	case op
		when '+' , 'Soma' , 'soma'
			puts "\nInsira o valor inicial"; a = gets.chomp.to_f
			puts "Insira o valor a ser somado"; b = gets.chomp.to_f 
			c = a+b; puts "O resultado é #{c}"
		
		
		when '-' , 'Multiplicação' , 'multiplicação' , 'Multiplicacao' , 'multiplicacao' 
			puts "\nInsira o valor inicial"; a = gets.chomp.to_f
			puts "Insira o valor a ser subitraido"; b = gets.chomp.to_f
			c = a-b; puts "O resultado é #{c}"

		when '*' , 'Divisão' , 'divisão' , 'Divisao' , 'divisao' 
			puts "\nInsira o fator inicial"; a = gets.chomp.to_f
			puts "Insira o segundo fator "; b = gets.chomp.to_f
			c = a*b; puts "O resultado é #{c}"

		when '/' , 'Subtração' , 'subtração' , 'Subtracao' , 'subtracao' 
			puts "\nInsira o dividendo"; a = gets.chomp.to_f
			puts "Insira o divisor"; b = gets.chomp.to_f
			c = a/b; puts "O resultado é #{c}"

		when '^' , 'Exponenciação' , 'exponenciação' , 'Exponenciacao' , 'exponenciacao' 
			puts "\nInsira o valor a ser fatorado"; a = gets.chomp.to_f
			puts "Insira o exponente"; b = gets.chomp.to_f
			c = a**b; puts "O valor de #{a} elevado à #{b} é #{c}"		

		when 'v' , 'Radiciação' , 'radiciação' , 'Radiciacao' , 'radiciacao' 
			puts "\nInsira o índice"; a = gets.chomp.to_f
			puts "Insira o radiciando"; b = gets.chomp.to_f
			c = b**(1/a); a = a.to_i;puts "A raiz #{a}a de #{b} é #{c}"	
		else
			puts "\nPor favor insira um operador válido"
		end
puts "Até mais!"
